src/app/shared/services/tissue-data/tissue-data.service.ts

```diff
@@ -58,4 +86,19 @@ export class TissueDataService {
   getOrganName(): Observable<string> {
     return this.activeTissue.pipe(pluck('slice', 'sample', 'patient', 'anatomicalLocations', 0));
   }
+
+  /**
+   * Finds the overlay that matches an ontology node.
+   *
+   * @param location The ontology node.
+   * @param overlays The overlays.
+   * @returns The matching overlay object or undefined if no match.
+   */
+  private findMatchingOverlay(location: OntologyNode, overlays: TissueOverlay[]): TissueOverlay {
+    return find(overlays, overlay => {
+      const locations = loMap(overlay.anatomicalLocations, toLower);
+      if (includes(locations, toLower(location.label))) { return true; }
```

Comment:
Should also check if there is a match on location.id in anatomicalLocations.

Reply:
Yep, forgot that. Done now!